sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/ScoringParameter.java

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Represents a parameter value to be used in scoring functions (for example, referencePointParameter).
+ */
+public final class ScoringParameter {
+    private final ClientLogger logger = new ClientLogger(ScoringParameter.class);
+    private final String name;
+    private final List<String> values;
+
+    private static final String SEPARATOR = "-";
```

Comment:
nit: Could this be renamed dash to more closely follow the `-` character's name.

Reply:
Renamed to DASH